---
 nm-ssh-service.name.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: b/nm-ssh-service.name.in
===================================================================
--- a/nm-ssh-service.name.in
+++ b/nm-ssh-service.name.in
@@ -4,7 +4,7 @@ service=org.freedesktop.NetworkManager.o
 program=@LIBEXECDIR@/nm-ssh-service
 
 [GNOME]
-auth-dialog=nm-ssh-auth-dialog
+auth-dialog=@LIBEXECDIR@/nm-ssh-auth-dialog
 properties=libnm-ssh-properties
 supports-external-ui-mode=true
 
